backend/Repositories/SightingRepo.cs

```diff
@@ -24,5 +27,25 @@ public IEnumerable<Sighting> GetAllSightings()
                 .Sightings
                 .Include(s => s.Species);
         }
+
+         public Sighting GetById(int id)
+        {
+            return _context
+                .Sightings
+                .Single(s => s.Id == id);
+        }
+         public Sighting DeleteSighting(int id)
```

Comment:
Line separating these two methods please!
```suggestion
        }
        
         public Sighting DeleteSighting(int id)
```